File: android/source/VideoLocker/src/org/edx/mobile/module/registration/model/RegistrationOption.java
package org.edx.mobile.module.registration.model;

import com.google.gson.annotations.SerializedName;

public class RegistrationOption {
    private @SerializedName("default")     boolean defaultValue;
    private @SerializedName("name")     String name;
    private @SerializedName("value")    String value;

    public String getName() {
        return name;
    }

    public String getValue() {
        return value;
    }

    public boolean isDefaultValue() {
        return defaultValue;
    }

    @Override
    public String toString() {
        return name;
    }
}

File: OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/model/RegistrationOption.kt
package org.edx.mobile.module.registration.model

import com.google.gson.annotations.SerializedName

data class RegistrationOption(
    @SerializedName("default")
    var isDefaultValue: Boolean = false,

    @SerializedName("name")
    var name: String = "",

    @SerializedName("value")
    val value: String = "",
) {
    override fun toString(): String {
        return name
    }
}
